.circleci/config.yml

```diff
@@ -65,6 +66,34 @@ jobs:
           name: Run GraphQL Tests
           command: cd packages/graphql && yarn test
 
+  GraphQLEngineSchemaCheck:
+    <<: *defaults
+    working_directory: ~/origin
+    steps:
+      - attach_workspace:
+          at: .
+      - run: sudo npm install -g apollo
+      - run:
+          name: Starting server
+          command: npm run start --prefix packages/graphql
+          background: true
+      - run: sleep 5
+      - run: apollo service:check --endpoint=http://localhost:4002/
+
+  GraphQLEngineSchemaUpload:
+    <<: *defaults
+    working_directory: ~/origin
+    steps:
+      - attach_workspace:
+          at: .
+      - run: sudo npm install -g apollo
+      - run:
+          name: Starting server
+          command: npm run start --prefix packages/graphql
+          background: true
+      - run: sleep 5
+      - run: apollo service:push --tag=master --endpoint=http://localhost:4002
```

Comment:
Is it ok to push a new schema on every PR and every unit test run even if the schema hasn't changed ?

Reply:
The upload only currently happens on master.  If it makes sense, maybe we can add staging and stable later, but no, it doens't make sense to do the upload on every random branch.  See the filters:

https://github.com/OriginProtocol/origin/blob/mikeshultz/apollo-engine/.circleci/config.yml#L265-L268

The breaking change checks do, however, run on every branch just the same as any other test.